chart/values.schema.json

```diff
@@ -516,16 +516,7 @@
             "description": "Airflow executor.",
             "type": "string",
             "x-docsSection": "Common",
-            "default": "CeleryExecutor",
-            "enum": [
-                "LocalExecutor",
-                "LocalKubernetesExecutor",
-                "CeleryExecutor",
-                "KubernetesExecutor",
-                "CeleryKubernetesExecutor",
-                "airflow.providers.amazon.aws.executors.batch.AwsBatchExecutor",
-                "airflow.providers.amazon.aws.executors.ecs.AwsEcsExecutor"
-            ]
+            "default": "CeleryExecutor"
```

Comment:
What about adding two new values (executors which is an array of executors, and defaultExecutor) and deprecate this one 🤔

Reply:
What if we keep this as a list of enums? Then, in the value, we concatenate them with `,` into a comma separated list? Downside is, this is still a fixed list of executors. You cannot specify custom executors or use short names such as `airflow.providers.amazon.aws.executors.ecs.AwsEcsExecutor:AwsEcsExecutor`.